backend/open_webui/tasks.py

```
@@ -2,16 +2,87 @@
 import asyncio
 from typing import Dict
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)